snap-css-web/tests/e2e/testData.js

```diff
@@ -0,0 +1,86 @@
+// Load CSS
```

Comment:
would it not be better to use fixtures?

Reply:
Yeah I will change them to fixture to match the standard